bundles/binding/org.openhab.binding.bticino/src/main/java/org/openhab/binding/bticino/internal/BticinoDevice.java

```diff
@@ -264,30 +465,293 @@ else if (p_protocol_read.getProperty("messageType").equalsIgnoreCase("CEN Basic
+                                        .equalsIgnoreCase("Remote Control disabled")) {
+                                    state = 0;
+                                    eventPublisher.postUpdate(l_item.getName(),
+                                            (state == 1) ? OpenClosedType.OPEN : OpenClosedType.CLOSED);
+                                } else if (p_protocol_read.getProperty("messageDescription")
+                                        .equalsIgnoreCase("Remote Control enabled")) {
+                                    state = 1;
+                                    eventPublisher.postUpdate(l_item.getName(),
+                                            (state == 1) ? OpenClosedType.OPEN : OpenClosedType.CLOSED);
+                                }
+                                logger.debug("T_ControlRemote : {} / {}", l_item.getName(), state);
+                            }
+                            break;
+                    }
+                }
+            } else if (l_item instanceof StringItem) {
+                logger.debug("Gateway [{}], RECEIVED EVENT FOR StringItem [{}], TRANSLATE TO OPENHAB BUS EVENT",
+                        m_gateway_id, l_item.getName());
 
-                    if (p_protocol_read.getProperty("messageDescription").equalsIgnoreCase("Temperature value")) {
-                        eventPublisher.postUpdate(l_item.getName(),
-                                DecimalType.valueOf(p_protocol_read.getProperty("temperature")));
+                // THERMOREGULATION
+                // to be used for Homekit
+                if (p_protocol_read.getProperty("messageType").equalsIgnoreCase("thermoregulation")) {
+                    String state = null;
+                    switch (Integer.parseInt(l_binding_config.what)) {
+                        case 20:
+                            // Status Actuators
+                            if (p_protocol_read.getProperty("what").equalsIgnoreCase(l_binding_config.what)) {
+                                if (p_protocol_read.getProperty("messageDescription").equalsIgnoreCase("Heating OFF")) {
+                                    state = "Off";
+                                    eventPublisher.postUpdate(l_item.getName(), StringType.valueOf(state));
+                                } else if (p_protocol_read.getProperty("messageDescription")
+                                        .equalsIgnoreCase("Heating ON")) {
+                                    state = "HeatOn";
+                                    eventPublisher.postUpdate(l_item.getName(), StringType.valueOf(state));
+                                }
+                                logger.debug("T_Relay : {} / {}", l_item.getName(), state);
+                            }
+                            break;
+                        case 101:
+                            // Status Control Mode
+                            if (p_protocol_read.getProperty("hStatus").equalsIgnoreCase("2")) {
```

Comment:
No point in using `equalsIgnoreCase()` here

Reply:
OK
see commit https://github.com/openhab/openhab1-addons/pull/5486/commits/c1c8f3a86ba8641c7333467de21a0ba489322073